src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -378,17 +381,27 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
                         .onlyOnPhone,
                     ),
                   )}
-
-                <View style={[styles.smallTopMargin, {flexDirection: 'row'}]}>
-                  <Info
-                    height={theme.icon.size.normal}
-                    width={theme.icon.size.normal}
-                    style={{marginRight: theme.spacings.small}}
+                {isCarnet ? (
+                  <GlobalMessage
```

Comment:
I think we should rather add a global message context not just for carnet, and if we want a message for just carnet we need to specify that in the rules.